### ISSUE TITLE
More HTML/CSS/Redux Questions are needed

### ISSUE THREAD

**Issue by codemyjourney** [Inter-Camp/InterCamp#71]

**Comment by nb-0310** [comment]
Is this still open? I could work on some HTML, CSS, Javascript, React questions. But I navigated to the live heroku app and it seems to be inactive now.

![image](https://user-images.githubusercontent.com/75498623/228922846-efabab5a-cff8-4021-b9f1-b6e18c9b557f.png)

**Comment by stacykutyepov** [comment]
Hello. Yes, I’d need to activate it. Are you still interested on working on it?

Best regards,
Stacy

On Mar 30, 2023, at 1:56 PM, nb-0310 ***@***.***> wrote:

﻿

Is this still open? I could work on some HTML, CSS, Javascript, React questions. But I navigated to the live heroku app and it seems to be inactive now.

[image]<https://user-images.githubusercontent.com/75498623/228922846-efabab5a-cff8-4021-b9f1-b6e18c9b557f.png>

**Comment by nb-0310** [comment]
yeah sure i'll just fork the repo and start working on it. should i create seperate PRs for Issue #71 and Issue #69 as #71 would have HTML CSS Redux questions and #69 would have js questions?
